### ISSUE TITLE
action for pin selected on map component

### ISSUE THREAD

**Issue by maks** [Jasonette/documentation#64]
Following on from [this issue](https://github.com/Jasonette/JASONETTE-Android/issues/79) another useful feature for the Map component would be to have an action for when a specific pin is selected (clicked) on a map.

**Comment by arifrifai** [comment]
Agree.

**Comment by gliechtenstein** [comment]
Agreed! I think it's a matter of attaching `href` and `action` to the pins. In fact it sounds like an obvious feature that's missing! Just created issues:

https://github.com/Jasonette/JASONETTE-Android/issues/144

https://github.com/Jasonette/JASONETTE-iOS/issues/242
